.github/workflows/ci.yaml

```diff
@@ -0,0 +1,48 @@
+name: ci
+on:
+  push:
+    branches:
+      - master
+      - develop
```

Comment:
Can we limit this to the master branch only?

Reply:
This should probably be `release/**', `hotfix/**`, and maybe `develop`.  This will allow us to build clients for a given release/hotfix branch.  Then, once the release/hotfix goes out, we can merge the branch down to `master`.  

Including `develop` would allow us to have testable artifacts after every merge, which sounds nice in theory, but could cause some bloat issues.  (see my comment about upload).  

`master` should be excluded here, as we don't want to overwrite the artifacts that actual got released.

Reply:
probably `develop` master is out of sync

Reply:
Oh wow yeah it is.